### PR TITLE
Modify rule S6466: LaYC format

### PR DESCRIPTION
**Pull request by joke1196**
## Review

A dedicated reviewer checked the rule description successfully for:

- [ ] logical errors and incorrect information
- [ ] information gaps and missing content
- [ ] text style and tone
- [ ] PR summary and labels follow [the guidelines](https://github.com/SonarSource/rspec/#to-modify-an-existing-rule)

### CODE REVIEW

File: rules/S6466/python/rule.adoc [hunk 1]
```diff
@@ -1,27 +1,37 @@
...
     ls = [1, 2, 3]
     foo(ls[2])
 
 ----
 
+== Resources
+
+=== Documentation
+
+* Python Documentation - https://docs.python.org/3/library/exceptions.html#IndexError[IndexError]
```
Comment (joke1196) [inline rules/S6466/python/rule.adoc:33]: Was not 100% sure about the documentation. Is it really needed?

Comment (guillaume-dequenne-sonarsource) [reply rules/S6466/python/rule.adoc:33]: I don't think it really is, but now that it's there, I don't think we need to remove it 😄

**Review by guillaume-dequenne-sonarsource — APPROVED** [review]
LGTM

File: rules/S6466/python/rule.adoc [hunk 2]
```diff
@@ -1,27 +1,37 @@
+This rule raises an issue when trying to access a list or a tuple index that is out of bound.
```
Comment (jean-jimbo-sonarsource) [inline rules/S6466/python/rule.adoc:1]: "...out of bounds"
is the correct phrase

**Pull request merged**